source/lookup_table.hpp

```diff
@@ -98,21 +79,19 @@ class AddressLookupTable
 		}
 
 		constexpr UINT CacheIndex = AddressCacheIndex<T>::CacheIndex;
-
-		for (size_t i = 0; i < AddressCache[CacheIndex].size(); i++)
+		for (auto it = AddressCache[CacheIndex].begin(); it != AddressCache[CacheIndex].end(); ++it)
```

Comment:
Fairly sure this could be replaced with `std::find_if` and an appropriate predicate.

Reply:
Good idea!  Updated code to use find_if rather than a for loop.